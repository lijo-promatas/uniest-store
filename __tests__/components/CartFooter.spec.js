import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import CartFooter from '../../src/components/CartFooter';

describe('<CartFooter />', () => {
  it('renders <CartFooter /> component correctly', () => {
    const tree = renderer.create(<CartFooter
      totalPrice="100%"
      btnText="test"
      onBtnPress={() => {}}
      isBtnDisabled={false}
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
