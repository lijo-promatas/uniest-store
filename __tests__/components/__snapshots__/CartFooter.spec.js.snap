// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartFooter /> renders <CartFooter /> component correctly 1`] = `
<View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      TOTAL
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      100%
    </Text>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        test
      </Text>
    </Text>
  </View>
</View>
`;
